pkgs/by-name/po/powersploit/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, fetchFromGitHub
+, stdenv
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "powersploit";
+  version = "3.0.0";
+
+  src = fetchFromGitHub {
+    owner = "PowerShellMafia";
+    repo = "PowerSploit";
+    rev = finalAttrs.version;
```

Comment:
```suggestion
    rev = "v${finalAttrs.version}";
```

Reply:
Ah, my bad, forgot that my hash was already cached when I ran the expression to test. Thanks.